examples/data_preprocessing/MIMIC_III_dataset/[WIP]_prepare_MIMIC_III_data.ipynb

```diff
@@ -0,0 +1,1340 @@
+    "\n",
+    "However, it is only available for credentialed users that sign the data use agreement. \n",
+    "\n",
+    "### PhysioNet credentialing\n",
+    "\n",
+    "In order to become a credentialed user, one needs to create a PhysioNet account and complete the credentialing process. This involves passing the \"Data or Specimens Only Research\" course from the \"Human Research\" curriculum offered by the Collaborative Institutional Training Initiative (CITI Program). For details, please create an account at https://physionet.org/ and https://about.citiprogram.org/ and follow the instructions on https://physionet.org/login/?next=/settings/credentialing/.\n",
+    "\n",
+    "### Data download\n",
+    "\n",
+    "Once your PhysioNet application is approved, there are no more obstacles in the way which could prevent you from downloading the data. The needed datasets can be downloaded from https://physionet.org/content/mimiciii/1.4/ (be aware that one needs to be signed in with their PhysioNet account). All tables available for download are listed on the bottom of this page. For this tutorial, we will need 3 datasets: DIAGNOSES_ICD.csv, D_ICD_DIAGNOSES.csv and NOTEEVENTS.csv:\n",
+    "\n",
+    "![files_1.PNG](attachment:files_1.PNG)\n",
+    "![files_2.PNG](attachment:files_2.PNG)\n",
+    "\n",
+    "In order to be able to run this notebook, please download these datasets and save the .csv files in your working directory."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data description"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "MIMIC-III is a relational database consisting of 26 tables. Tables are linked by identifiers which usually have the suffix ‘ID’. For example, SUBJECT_ID refers to a unique patient, HADM_ID refers to a unique admission to the hospital, and ICUSTAY_ID refers to a unique admission to an intensive care unit.\n",
+    "\n",
+    "Charted events such as notes, laboratory tests, and fluid balance are stored in a series of ‘events’ tables. For example the NOTEEVENTS table contains all clinical notes related to an event for a given patient.\n",
+    "\n",
+    "Tables prefixed with ‘D_’ are dictionary tables and provide definitions for identifiers. For example, the D_ICD_DIAGNOSES table provides definitions for each ICD9 Code, which are used to indicate the diagnoses in the DIAGNOSES_ICD table.\n",
+    "\n",
+    "For the purpose of this project, we will focus on the clinical notes, stored as text in NOTEEVENTS and corresponding diagnoses saved in DIAGNOSES_ICD and D_ICD_DIAGNOSES."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports\n",
+    "\n",
+    "First, let's make some basic imports."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
```

Comment:
the imports should be ordered according to the style guide, see https://www.python.org/dev/peps/pep-0008/#id21
(you can use hotkeys to order them automatically, e.g. in PyCharm: https://www.jetbrains.com/help/pycharm/creating-and-optimizing-imports.html#optimize-imports

Reply:
I hope they are ordered correctly now.